packages/util/package.json

```diff
@@ -35,12 +35,12 @@
     },
     "dependencies": {
         "@giphy/js-types": "*",
-        "dompurify": "^2.2.2",
+        "dompurify": "2.2.2",
```

Comment:
@nicksantan maybe just upgrade this to the latest since we'll have to test anyway